glib-macros/src/lib.rs

```diff
@@ -310,3 +313,76 @@ pub fn gflags(attr: TokenStream, item: TokenStream) -> TokenStream {
     let gen = gflags_attribute::impl_gflags(&input, &gtype_name);
     gen.into()
 }
+
+/// Macro for deriving implementations of [`glib::Downgrade`] and
+/// [`glib::Upgrade`] traits and a weak type.
+///
+/// # Examples
+///
+/// ## New Type Idiom
+///
+/// ```rust,ignore
+/// #[derive(Downgrade)]
```

Comment:
Isn't it supposed to be `#[derive(glib::Downgrade)]`?

Reply:
Done